BookingsApi/BookingsApi.Contract/Helper/HearingScottishVenueNames.cs

```diff
@@ -30,6 +43,18 @@ public static class HearingScottishVenueNames
         InvernessJusticeCentre,
         EdinburghSocialSecurityAndChildSupportTribunal,
         EdinburghUpperTribunal,
+        Campbeltown,
```

Comment:
I think you're missing Benbecula, from looking at the spreadsheet in the ticket

Reply:
well spotted. I have added it